shell/platform/fuchsia/flutter/compositor_context.cc

```diff
@@ -58,19 +59,27 @@ class ScopedFrame final : public flutter::CompositorContext::ScopedFrame {
       // represent the frame are enqueued in the underlying session.
       TRACE_EVENT0("flutter", "UpdateScene");
       layer_tree.UpdateScene(scene_update_context_);
+      frame_paint_tasks = scene_update_context_->GetPaintTasks();
```

Comment:
What's the reason to move this?

Reply:
Hygiene, but I'll put that in the follow-on CL with tests